src/main/java/net/dries007/tfc/objects/items/ItemsTFC.java

```diff
@@ -321,6 +325,15 @@ public static void registerItems(RegistryEvent.Register<Item> event)
         // todo: foods & plants & seeds & fruits & fruit tree saplings & berries & berry bushes
 
         allSimpleItems = simpleItems.build();
+        // Register oredict bronze items as any bronze
```

Comment:
This should either be in the `ItemMetal` constructor, or if it really needs to be separate, in `OreDictionaryHelper#init` imo. Preferably the former.